File: backend/Domains/Question.cs
namespace LumiLearn.Domains
{
    public class Question
    {
        public Guid Id { get; set; }
        public string Content { get; set; }
        public Guid QuizId { get; set; }

        // Navigation Properties
        public Quiz Quiz { get; set; }
        
        public ICollection<AnswerOption> AnswerOptions { get; set; }
    }
}

File: frontend/types/question.ts
import { AnswerOptionContent, AnswerOptionDto } from "./answerOption";

export type QuizQuestionDto = {
    id: string;
    content: string;
    answerOptions: AnswerOptionDto[];
}

export type QuestionWithContent = {
    content: string;
    answerOptions: AnswerOptionContent[];
};